PowerEditor/installer/nativeLang/Malayalam.xml

```diff
@@ -0,0 +1,954 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+***********************************************************************************
+Malayalam language binary translations file for Notepad++ ::
+Created By:- Roshan Kishorbhai Rathod
+Email id- rathodroshan007@gmail.com
+College Name: Shrimad Rajchandra Institute of Management & Computer Application(SRIMCA) 
+************************************************************************************
+	
+-->
+<NotepadPlus>
+    <Native-Langue name="ഇംഗ്ലീഷ്" filename="Malayalam.xml" version="6.8.2">

```

Comment:
@roshan0 latest version of notepad is 7.5.8. I assume that you have referred latest English.xml and translated accordingly. If so, then change this version here too.

Reply:
@SinghRajenM  yes

Reply:
Thanks wait for new version release

Reply:
@SinghRajenM  Done Buddy